src/app/playlist/playlist.component.html

```diff
@@ -1,5 +1,6 @@
 <div class="row">
-  <div class="col-sm-3">
```

Comment:
Why did you change the column layout size? We want this layout to persist to smaller screens. Then we can make a separate layout format for mobile.